src/pages/HomePage.vue

```diff
@@ -2,46 +2,53 @@
   <q-page class="flex column justify-between home-page">
     <div class="fit flex justify-center items-center home-banner">
       <q-img
-          src="icons/logo_modelizer.svg"
-          class="home-logo"
-          :ratio="1"
+        src="icons/logo_modelizer.svg"
+        class="home-logo"
+        :ratio="1"
       />
     </div>
-    <div class="flex home-content">
+    <div class="column items-center home-content">
       <q-btn
-          color="primary"
-          size="xl"
-          icon="fa-solid fa-circle-plus"
-          data-cy="createEmptyProject"
-          :label="$t('actions.home.createEmptyProject')"
-          :to="`/modelizer/${project.id}/model`"
+        color="primary"
+        size="xl"
+        icon="fa-solid fa-circle-plus"
+        data-cy="create-empty-project"
+        :label="$t('actions.home.createEmptyProject')"
+        :to="`/modelizer/${project.id}/model`"
+        @click="updateProject(project)"
       />
+      <div class="fit row justify-center">
+        <ProjectGrid
+          class="col-md-8"
+          :projects="getProjects()"
```

Comment:
Why use a prop instead of directly get it in the component ProjectGrid ?

Reply:
It allows testing by injecting data more easily than passing through the localStorage. Though when I tried to setup the unit test for this component I encountered another issue with the `i18n` layer not injected in the test runtime and the plugin for doing that in just is currently outdated. So I ended up not writing the test.

http://blog.ingeniouscontraptions.com/2020/12/10/integrer-vue-i18n-dans-les-tests-jest-dun-projet-vue-js/